packages/training/deploy/docker/scripts/validate.sh

```
@@ -0,0 +1,133 @@
+        print(f'  ✓ Connected - {count} training trajectories')
+    await pool.close()
+asyncio.run(test())
+" || { echo "  ✗ Database connection failed"; ERRORS=$((ERRORS + 1)); }
+fi
+
+# 5. GPU profiles
+echo ""
+echo "[5/6] Checking GPU profiles..."
+PROFILE_DIR="/app/python/config/profiles"
+if [ -d "$PROFILE_DIR" ]; then
+    PROFILE_COUNT=$(ls -1 "$PROFILE_DIR"/*.json 2>/dev/null | wc -l)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)